src/docs/pages/index.jsx

```diff
@@ -151,6 +151,11 @@ const index = () => (
       </a>.
     </p>
 
+    <input
+      type="hidden"
+      value="https://styleguide.brainly.com/%%images/icons.svg%%"
+    />
```

Comment:
What is this `input` for? 🤔

Reply:
it could be any hidden element, i need this value to be available somehow to use it later on.

Reply:
Ah, it's just to know the url?

How are you going to test it? Does it need to be served from style-guide? If it's just local testing it sounds like it could be also locally served.